app/models/review.model.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (sequelize, Sequelize) => {
+    const Review = sequelize.define("review", {
+        review:{
+            type: Sequelize.STRING,
+            allowNull: false,
+        }
```

Comment:
just a thought, but I wonder if it would make sense to keep a date/time on each review which gets set to the current date/time when the review is created, might be helpful to the user to be able to see reviews sorted by recency

Reply:
Time stamps are added by sequelize